content/200-concepts/300-more/050-environment-variables/index.mdx

```diff
@@ -7,18 +7,29 @@ tocDepth: 2
 
 <TopBlock>
 
-You can use environment variables to provide configuration options when a CLI command is invoked. Environment variables allow you to:
+An environment variable is a key value pair stored on your machines local environment. Traditionally the name of the variable is uppercase, this is then followed by an equals sign then the value of the variable.
 
-- Keep secrets out of the schema file
-- Improve portability of the schema file
-
-You can use machine-specific variables, `.env` files, or a combination of both.
+```env
+MY_VALUE=prisma
+```
 
 </TopBlock>
 
+## How does Prisma use environment variables?
+
+Prisma always reads environment variables from the systems environment. 
+
+When you install Prisma for the first time it creates a convenience `.env` file for you to set your [`connection url`](../../reference/database-reference/connection-urls) as an environment variable. The `.env` file puts the variable in the systems environment and Prisma reads it from there.
+
+<Admonition type="info">
+
+Looking to use more than one `.env` file? See [Using multiple `.env` files](./using-multiple-env-files) for information on how to setup and use multiple `.env` files in your application.
+
+</Admonition>
+
 ## Using .env files
 
-For many developer tools, it has become a good practice to define environment variables using [`.env`](https://github.com/motdotla/dotenv) files. The Prisma CLI, and Client, looks for `.env` files, in order, in the following locations:
+Although Prisma creates a default `.env` file at your projects root, you can create additional (or replace) `.env` files at any location in your projects filesystem. The Prisma CLI looks for `.env` files, in order, in the following locations:
```

Comment:
"additional" is the multiple files use case, this really is about the user making a choice and how Prisma finds the file. The locations here are limited: root, prisma folder, next to your schema. Not anywhere in the file system.